packages/client/modules/teamDashboard/components/AgendaList/AgendaList.tsx

```diff
@@ -124,18 +125,12 @@ const AgendaList = (props: Props) => {
 }
 
 export default createFragmentContainer(AgendaList, {
-  team: graphql`
-    fragment AgendaList_team on Team {
-      agendaItems {
-        id
-        content
-        # need this for the DnD
-        sortOrder
-        ...AgendaItem_agendaItem
-      }
-      activeMeetings {
-        ...AgendaItem_activeMeetings
-        id
+  meeting: graphql`
```

Comment:
if the component isn't directly using a property, it shouldn't be requesting it.
e.g. `endedAt` only appears once in this file-- in the graphql query. therefore, it shouldn't exist at all